cloudfront/wellcomelibrary.org/edge-lambda/testRedirects.ts

```diff
@@ -1,24 +1,23 @@
-import path from 'path';
-import fs from 'fs';
 import axios from 'axios';
 import chalk from 'chalk';
 
-type RedirectTest = {
-  host: string;
-  pathTests: Record<string, string>;
-};
-
-type HostEnvs = {
-  prod: string;
-  stage: string;
-};
+import { readRedirects } from './src/readRedirects';
+import {
+  staticRedirectFileLocation,
+  staticRedirectHeaders,
+  staticRedirectsHost,
+} from './staticRedirects';
 
+type CsvHeader = string | undefined;
 type EnvId = 'stage' | 'prod';
 
-type RedirectTestSet = (env: EnvId) => RedirectTest;
+type HostEnvs = {
+  prod?: string;
+  stage?: string;
+};
```

Comment:
You can do something like [this](https://www.typescriptlang.org/play?ssl=10&ssc=33&pln=1&pc=1#code/C4TwDgpgBAEg9gZ2AUQHYDcFQLxQN4CwAUFFGAE5wAmA-AFxRLkCWqA5gNzGlICGbEeo2At2XIgF9xoSFDToAklRxQA1hBBwAZrEQoMCYsQDGcVEjKUqDeUpUByCtXsmzFvgJsY7uex4guRKbmwFAARnBhYQA2EKjUCF6Kyr4RUbHxVAiBwRaoAK5aWqxsST5Q+ahUEMWoEFRAA) too (typescripts lovely ability to create types from types)